examples/illustrations/call_log/call_log.json

```diff
@@ -201,11 +201,11 @@
         },
         {
             "@id": "kb:conference-bridge-phone-call-1",
-            "@type": "uco-observable:PhoneCall",
+            "@type": "uco-observable:Call",
```

Comment:
Until 0.8.0 is released, these should be in the `drafting` namespace, taking advantage of `drafting.ttl` to reduce validation errors.